docs_sphinx/user/nmlimport.rst

```diff
@@ -0,0 +1,322 @@
+NeuroML importer (nmlimport library)
+================
+
+This is a short overview of the `~brian2tools.nmlimport` package, providing
+functionality to import Brian's morphology from a .nml file.
+
+NeuroML is a XML-based description that provides a common data format
+for defining and exchanging descriptions of neuronal cell and network models
+(`NML project website <https://neuroml.org/>`_).
+
+.. contents::
+    Overview
+    :local:
+
+Working example
+---------------
+
+As a demonstration, we will use nmlimport library to generate morphology and
+extract other related information from the ``pyramidal cell's`` nml file. You
```

Comment:
Putting "pyramidal cell's" in verbatim looks a bit odd here -- it would make sense if you used the actual file name, though (`pyr_4_sym.cell.nl`).